frontend/src/metabase/core/components/Alert/Alert.tsx

```diff
@@ -22,7 +22,12 @@ const Alert = ({
   className,
 }: AlertProps) => {
   return (
-    <AlertRoot hasBorder={hasBorder} className={className} variant={variant}>
+    <AlertRoot
+      hasBorder={hasBorder}
+      className={className}
+      variant={variant}
+      role="alert"
```

Comment:
based on Mantine's `Alert` it also has the role `alert`, this is so that I could use query `*ByRole("alert")` in the tests.